lib/quantum.ex

```diff
@@ -51,22 +66,8 @@ defmodule Quantum do
       config
       |> Keyword.get(:jobs, [])
       |> Enum.map(&Normalizer.normalize(quantum.new_job(config), &1))
-      |> remove_jobs_with_duplicate_names(quantum)
+      |> Enum.uniq_by(fn %Job{name: name} -> name end)
```

Comment:
I think it's worth a warning. OTP20 is warning about duplicate keys in literal maps now. I say we should warn about duplicate jobs. This will eliminate quite a lot programmer error (or hopefully not that many)